index.html

```diff
@@ -218,41 +234,43 @@ <h2 class="project-title">Multi-Post Stories</h2>
-               </p>
+         <section class="contact" id="contact">
+            <div class="contact-container">
+               <div class="contact-text">
+                  <h1 class="contact-title">
+                     Let's chat
+                  </h1>
+                  <p class="contact-description">
+                     If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+                  </p>
+               </div>
+   
+               <form class="contact-form" action="https://formspree.io/f/xnqwkjoy" method="post">
+                  <ul>
+                     <li>
+                        <label for="user-name"></label>
+                        <input type="text" placeholder="Enter your name" class="user-name" id="user-name" name="user-name" maxlength="30" required >
+                     </li>
+                     <li>
+                        <label for="email"></label>
+                        <input type="email" name="email" id="email" placeholder="Email" class="user-email" required >
+   
+                     </li>
+                     <li>
+                        <label for="user-text"></label>
+                        <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Write your message here" maxlength="500" required></textarea>
+   
+                     </li>
+                     <li>
+                        <button class="see-projects project-btn" type="submit" value="send">
+                           Send
+                        </button>
+   
+                     </li>
+                  </ul>
+                
+               </form>
             </div>
-
-            <form class="contact-form" action="https://formspree.io/f/xnqwkjoy" method="post">
-               <ul>
-                  <li>
-                     <label for="user-name"></label>
-                     <input type="text" placeholder="Enter your name" class="user-name" id="user-name" name="user-name" maxlength="30" required >
-                  </li>
-                  <li>
-                     <label for="email"></label>
-                     <input type="email" name="email" id="email" placeholder="Email" class="user-email" required >
-
-                  </li>
-                  <li>
-                     <label for="user-text"></label>
-                     <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Enter text here" maxlength="500" required></textarea>
-
-                  </li>
-                  <li>
-                     <button class="see-projects project-btn" type="submit" value="send">
-                        See Project
-                     </button>
-
-                  </li>
-               </ul>
-             
-            </form>
          </section>
 
          <!-- footer -->
```

Comment:
Great work @John-Kibirige 🤝

- [ ] To make your design a total replica of the Figma design, you may as well add the dark line just above your footer section. If you take a close look at the Figma design, you will notice a dark line. 
- Figma design
![footer figma](https://user-images.githubusercontent.com/63917725/177809657-a6121e35-5a82-4e01-9e0a-50db7bc90218.PNG)